compiler/noirc_evaluator/src/brillig/brillig_gen/brillig_block.rs

```diff
@@ -838,12 +832,21 @@ impl<'block, Registers: RegisterAllocator> BrilligBlock<'block, Registers> {
                     dfg,
                 );
 
+                let offseted = if dfg.is_constant(*index) {
```

Comment:
```suggestion
                let has_offset = if dfg.is_constant(*index) {
```
Let's change this from `offseted` which triggers cspell

Reply:
Done!